launcher/ui/instanceview/AccessibleInstanceView.cpp

```diff
@@ -298,8 +298,12 @@ bool AccessibleInstanceView::unselectRow(int row)
                 //the ones which are down the current row will be deselected
                 selection = QItemSelection(index, view()->model()->index(rowCount() - 1, 0, view()->rootIndex()));
             }
+            break;
         }
         default: {
+            // FIXME: See if MultiSelection / ExtendedSelection need to be handled
+            if (selectionType != QAbstractItemView::NoSelection)
+                qWarning() << "Unhandled QAbstractItemView selection type!";
             break;
         }
```

Comment:
What do you think about just breaking on ones that we know we should ignore instead of using an if statement to exclude them from the default case?
```suggestion
        }
        case QAbstractItemView::NoSelection:
            break;
        default: {
            // FIXME: See if MultiSelection / ExtendedSelection need to be handled
            qWarning() << "Unhandled QAbstractItemView selection type!";
            break;
        }
```